doc/sphinx/index.rst

```diff
@@ -1,24 +1,107 @@
-GPflow Documentation
-==================================
+GPflow
+======
 
-.. toctree::
-   :maxdepth: 1
-   :caption: Intro:
+GPflow is a package for building Gaussian Process models in python, using `TensorFlow
+<http://www.tensorflow.org>`_. Gaussian Processes are a kind of supervised learning models.
```

Comment:
```suggestion
<http://www.tensorflow.org>`_. A Gaussian Process is a kind of supervised learning model.
```
not sure if this is the best rewording, but currently there is a bit of a mismatch between singular & plural.

Reply:
> not sure if this is the best rewording, but currently there is a bit of a mismatch between singular & plural.

Done.